deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml

```diff
@@ -34,7 +34,7 @@ metadata:
             "namespace": "default"
           },
           "spec": {
-            "image": "registry.connect.redhat.com/storageos/cluster-operator:1.0.0",
+            "image": "registry.connect.redhat.com/storageos/cluster-operator:latest",
```

Comment:
Curious why it uses a mix of fixed version and latest?

Reply:
This is part of the example we provide. We don't want to be updating the example for every release.